vassal-app/src/main/java/VASSAL/build/module/chessclockcontrol/ChessClock.java

```diff
@@ -53,16 +53,9 @@
 import VASSAL.command.Command;
 import VASSAL.command.CommandEncoder;
 import VASSAL.command.NullCommand;
-import VASSAL.configure.ColorConfigurer;
-import VASSAL.configure.ComponentDescription;
-import VASSAL.configure.Configurer;
-import VASSAL.configure.ConfigurerFactory;
-import VASSAL.configure.IconConfigurer;
-import VASSAL.configure.TranslatableStringEnum;
+import VASSAL.configure.*;
```

Comment:
I think our style-thing doesn't let us use .* imports (I guess the auto-build will probably soon tell you that)